db/seeds.rb

```diff
@@ -76,13 +76,15 @@ def generate_dogs
   end
 
   def generate_observations
-    25.times do |i|
-      Observation.create(
-      observer_id: 2*i,
-      observed_id: (Dog.last + i),
-      liked: true,
-      distance: rand(9) )
-      puts "It's a match!"
+    50.times do |n|
+      25.times do |i|
+        Observation.create(
+        observer_id: 2*i,
```

Comment:
Indent the first parameter one step more than the previous line.<br>Surrounding space missing for operator '*'.